webserver/client/codechecker/cmdhelper/cmd_line_client.py

```diff
@@ -28,7 +28,7 @@
 
 from ..client.client import handle_auth, setup_client
 
-from libcodechecker import package_context
+from codechecker import server_context
```

Comment:
The package name is `webserver_context` not `server_context` right? There are other `server_context` imports too.